packages/appium/docs/zh/index.md

```diff
@@ -1,6 +1,21 @@
 ---
-title: Appium文档
+title: Appium文档首页
 ---
+![image](https://github.com/appium/appium/assets/48639567/9dbaf0a0-df6c-4d61-be3e-617653ee1065)
 
+欢迎来到Appium文档！Appium是一个开源项目及其相关软件生态系统，旨在促进许多应用平台的用户界面自动化，包括移动设备（iOS、Android、Tizen）、浏览器（Chrome、Firefox、Safari）、桌面（macOS、Windows）、电视（Roku、tvOS、Android TV、Samsung）等。
+如果您是第一次接触Appium，建议您从 [介绍](https://github.com/appium/appium) 
+开始，然后进入 [快速入门](https://github.com/appium/appium)。
+您随时可以在 [GitHub](https://github.com/appium/appium) 上找到Appium的代码。
+Appium主要用于软件测试自动化领域，以帮助确定给定应用程序的功能是否按预期工作。与其他类型的软件测试不同，UI自动化允许测试人员编写代码，模拟在应用程序的实际用户界面中执行用户场景，尽可能地模拟真实世界中发生的情况，同时实现自动化的各种好处，包括速度、规模和一致性。
+Appium旨在提供一套工具，以一种标准的方式支持跨任意数量的平台进行这种自动化。大多数平台都提供在某个层次上允许进行UI自动化的工具，但这些通常是特定于平台的，需要专业知识以及特定的编程语言经验和工具链。
+Appium试图将所有这些自动化技术统一到一个稳定的界面下，通过大多数流行的编程语言访问（您可以使用Java、Python、Ruby、JS等编写Appium脚本）。
+要了解Appium如何实现这一目标以及涉及的各种组件，请阅读我们的[介绍](https://appium.io/docs/en/2.2/intro/)。
```

Comment:
This and the other `appium.io` links are hardcoded to `2.2`, which is not good. For now just use the filepath to the EN docs (`../en/intro/index.md`)

Reply:
ok, thank you very much, I will correct it.